### PR TITLE
Move dashboard script

### PR DESCRIPTION
**Pull request by josephgregoryii**

### CODE REVIEW

File: utils/lib/dashboards.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+export type DashboardFileAndPath = FilePathAndContents<Dashboard>;
+
+export const getAllDashboardPaths = () => {
+  return findQuickstartComponentConfiguration('dashboards');
+};
+
+// .sjon
+// .png
+// assets/iconm.svg
```
Comment (zstix) [inline utils/lib/dashboards.ts:23]: Do we need these in here? If so, it might be good to add some more context (and fix some spelling mistakes 😄)

Comment (josephgregoryii) [reply utils/lib/dashboards.ts]: Removed and included documentation for functions

File: utils/migration_scripts/move-dashboards.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    );
+    const dashboardScreenshots = dashboardScreenshotPaths.map((screenshot) => {
+      return {
+        oldFilePath: screenshot,
+        newFilePath: path.join(newDashboardLocation, path.basename(screenshot)),
+      };
+    });
+    // create new files in this location
+    dashboardScreenshots.forEach(({ oldFilePath, newFilePath }) =>
+      fs.copyFileSync(oldFilePath, newFilePath)
```
Comment (aswanson-nr) [inline utils/migration_scripts/move-dashboards.ts:105]: Could probably use the `copyFiles` function here, not a required change at all.

**Review by aswanson-nr — APPROVED** [review]
Looking good to me!

**Review by tabathadelane — APPROVED** [review]

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**